### DISCUSSION TITLE
Feedback link is missing for failed searches

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#11826] [Other Feature Feedback, Questions, & Ideas] [answered]
https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Acheck-spelling+.*medium.*+path%3A.github%2Factions%2Fspelling

<img width="981" alt="image" src="https://user-images.githubusercontent.com/2119212/154861733-f3397550-6914-4be3-9560-6f5a23f64752.png">

https://cs.github.com/?scopeName=All+repos&scope=&q=org%3Acheck-spelling+%2F.*medium.*%2F+path%3A.github%2Factions%2Fspelling

<img width="975" alt="image" src="https://user-images.githubusercontent.com/2119212/154861751-e3f81475-937a-451a-8208-b2ba582ba586.png">

**Comment by colinwm** [comment, marked as answer]
Great point! I've added it to the failed search page as well as the no results page. Thanks for the feedback!

**Reply by jsoref** [reply]
Works in Chrome on Android when requesting desktop page. Not visible when getting mobile view.

![Screenshot_20220223-122735](https://user-images.githubusercontent.com/2119212/155373651-3fd6f55a-be98-4726-9b9e-11dccb8a1e1d.png)
